scripts/coderabbit_retry/coderabbit_retry.py

```
@@ -0,0 +1,331 @@
+                repo_name=repo_name,
+                repo=repo,
+                pr_number=pr_issue.number,
+            )
+            for pr_issue in to_process
+        ]
+        for future in as_completed(fs=futures):
+            try:
+                if future.result():
+                    retried += 1
+                    if retried >= MAX_RETRIES_PER_RUN:
+                        LOGGER.info(f"Retry cap reached ({MAX_RETRIES_PER_RUN}) — remaining results deferred")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)